packages/support/src/SupportServiceProvider.php

```diff
@@ -165,6 +126,22 @@ public function packageBooted(): void
 
                     return "<fg=red;options=bold>PUBLISHED:</> {$publishedViewPaths->join(', ')}";
                 },
+                'Blade Icons' => function (): string {
+                    return File::exists(app()->bootstrapPath('cache/blade-icons.php'))
+                        ? '<fg=green;options=bold>CACHED</>'
+                        : '<fg=yellow;options=bold>NOT CACHED</>';
+                },
+                'Form Components' => function (): string {
```

Comment:
The command caches panel components such as the autodiscovery of resources / pages / widgets, not form components

```suggestion
                'Panel Components' => function (): string {
```